src/crypto/mnemonic/mnemonic.cpp

```diff
@@ -1,10 +1,26 @@
 #include "crypto/mnemonic/mnemonic.h"
-#include <iostream>
+#include "crypto/sha256.h"
 #include <algorithm>
+#include <bitset>
+#include <sstream>
+#include <string>
 #include <boost/format.hpp>
 
 namespace mnemonic
 {
+    namespace
+    {
+        std::string binaryString(const uint8_t* data, const int& len)
```

Comment:
should be BinaryString. Our code convention means functions start with capital letters. Same for all the others below.